src/els_hover_provider.erl

```diff
@@ -74,23 +74,30 @@ documentation(_M, _POI) ->
 %% Future work: shell_docs:render/4 should probably be copied and
 %% modified to render markdown, or make an PR to OTP with an option
 %% in erlang's shell_docs to render markdown instead of ANSI-codes.
--spec get_docs(atom(), atom(), byte()) -> binary().
+%% -spec get_docs(atom(), atom(), byte()) -> binary().
+%% -spec get_docs(module(), function(), arity()) -> any().
+-spec get_docs(module(), function(), arity()) -> byte().
 -ifdef(OTP_RELEASE).
 -if(?OTP_RELEASE >= 23).
 get_docs(M, F, A) ->
-  case code:get_doc(M) of
-    {ok, #docs_v1{ format = ?NATIVE_FORMAT
-                 , module_doc = MDoc
-                 } = DocChunk} when MDoc =/= none ->
-      case shell_docs:render(M, F, A, DocChunk) of
-        {error, _R0} ->
-          docs_from_src(M, F, A);
-        FuncDoc ->
-          #{ kind => content_kind()
-           , value => els_utils:to_binary(FuncDoc)
-           }
-      end;
-    _R1 ->
+  try
+    case code:get_doc(M) of
+      {ok, #docs_v1{ format = ?NATIVE_FORMAT
+                   , module_doc = MDoc
+                   } = DocChunk} when MDoc =/= none ->
+        case shell_docs:render(M, F, A, DocChunk) of
+          {error, _R0} ->
+            docs_from_src(M, F, A);
+          FuncDoc ->
+            #{ kind => content_kind()
+             , value => els_utils:to_binary(FuncDoc)
+             }
+        end;
+      _R1 ->
+        docs_from_src(M, F, A)
+    end
+  catch C:E ->
+      lager:error("[hover] Error fetching EEP48 docs [error=~p]", [{C, E}]),
```

Comment:
This generates the log line 

```
2020-05-20 11:29:20.398 [error] <0.184.0>@els_hover_provider:get_docs:99 [hover] Error fetching EEP48 docs [error={error,{case_clause,{error,beam_lib,{file_error,"/usr/local/bin/erlang_ls/erlang_ls/ebin/els_hover_provider.beam",enotdir}}}}]
```

Reply:
This is the bug I was talking about. I am already addressing this in a local branch. Will push it today.

Reply:
And the related "echo, ok" error you will eventually encounter :)

Reply:
well, I will leave it to you then, good luck keeping dialyzer happy.  Will revert to what I was *actually* trying to do.

Reply:
Didn't mean to scare you away :) It's the group leader fix I mentioned at the weekly sync.

Reply:
Well, I was not scared away, per se, but just figured if you already have a branch, further along than mine, there is no point in us duplicating work.

Especially as my current interest is more toward the older versions anyway.